demisto_sdk/common/hook_validations/release_notes.py

```diff
@@ -0,0 +1,164 @@
+
+release_notes_standard = 'https://github.com/demisto/content/blob/master/docs/release_notes/README.md'
+
+
+class ReleaseNotesValidator:
+    """Release notes validator is designed to ensure the existence and correctness of the release notes in content repo.
+
+    Attributes:
+        file_path (str): the path to the file we are examining at the moment.
+
+    """
+
+    def __init__(self, file_path):
+        # type: (str) -> None
+        self.file_path = file_path
+        self.release_notes_path = get_release_notes_file_path(self.file_path)
+        self.release_notes = get_latest_release_notes_text(self.release_notes_path)
+        self.master_diff = self.get_master_diff()
+
+    def get_master_diff(self):
+        # type: () -> str
+        """Gets difference between current branch and origin/master
+
+        git diff with the --unified=10000 option means that if there exists a
+        difference between origin/master and current branch, the output will have at most 10000
+        lines of context.
+
+        Returns:
+            str. empty string if no changes made or no origin/master branch, otherwise full difference context.
+
+        """
+        try:
+            return run_command(F'git diff --unified=10000 '
+                               F'origin/master {self.release_notes_path}')  # type: str
+
+        finally:
+            return ''
+
+    def is_release_notes_changed(self):
+        # type: () -> bool
+        """Validates that a new comment was added to release notes.
+
+        Returns:
+            bool. True if comment was added, False otherwise.
+
+        """
+        # there exists a difference between origin/master and current branch
+        if self.master_diff:
+            diff_releases = self.master_diff.split('##')
+            unreleased_section = diff_releases[1]
+            unreleased_section_lines = unreleased_section.split('\n')
+
+            adds_in_diff = 0
+            removes_in_diff = 0
+
+            for line in unreleased_section_lines:
+                if line.startswith('+'):
+                    adds_in_diff += 1
+                elif line.startswith('-'):
+                    removes_in_diff += 1
+
+            # means that at least one new line was added
+            if adds_in_diff - removes_in_diff > 0:
+                return True
+
+        print_error(F'File {self.release_notes_path} is not formatted according to '
+                    F'release notes standards.\nFix according to {release_notes_standard}')
+        return False
+
+    def is_valid_release_notes_structure(self):
+        # type: () -> bool
+        """Validates that the release notes written in the correct manner.
+
+        one_line_release_notes_regex meaning:
+            dont start with any of the characters in the first []
+            starts with a letter or number
+            ends with '.'
+
+        multi_line_release_notes_regex meaning:
+            starts with tab, then '-' ,then space
```

Comment:
not sure about the tab, @yardensade could you share some insights?

Reply:
As I understand it, it's either a tab or something between 2 and 4 spaces.